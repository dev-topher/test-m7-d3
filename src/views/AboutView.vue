<!-- src/views/AboutView.vue -->
<template>
    <div>
      <h1>About Us</h1>
      <p>This is the About page.</p>
    </div>
  </template>
  
  <script>
  export default {
    name: 'AboutView',
  };
  </script>
  
  <style scoped>
  /* Estilos espec√≠ficos para AboutView.vue */
  </style>
  