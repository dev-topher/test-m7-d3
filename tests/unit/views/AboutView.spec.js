// tests/unit/views/AboutView.spec.js
import { shallowMount } from '@vue/test-utils';
import AboutView from '@/views/AboutView.vue';

describe('AboutView.vue', () => {
  it('renders the About page', () => {
    const wrapper = shallowMount(AboutView);
    expect(wrapper.text()).toContain('About Us');
    expect(wrapper.text()).toContain('This is the About page.');
  });

  it('matches the snapshot', () => {
    const wrapper = shallowMount(AboutView);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
