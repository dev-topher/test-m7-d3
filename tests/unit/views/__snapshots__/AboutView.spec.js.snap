// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutView.vue matches the snapshot 1`] = `
<div>
  <h1>About Us</h1>
  <p>This is the About page.</p>
</div>
`;
